bumpversion/__init__.py

```diff
@@ -183,7 +185,7 @@ def add_path(cls, path):
         pass
 
     @classmethod
-    def tag(cls, name, message):
+    def tag(cls, sign, name, message):
```

Comment:
I think this should raise an error when sign is true that it's not supported.

Reply:
👍 fixed.